platform/default/sqlite3.hpp

```diff
@@ -42,27 +48,23 @@ class Database {
 
     explicit operator bool() const;
 
-    static void errorLogCallback(void *arg, const int err, const char *msg);
     void setBusyTimeout(std::chrono::milliseconds);
     void exec(const std::string &sql);
     Statement prepare(const char *query);
 
-    int64_t lastInsertRowid() const;
```

Comment:
> Qt SQLite driver does not cache that - [it calls for `sqlite3_changes` and `sqlite3_last_insert_rowid` directly](https://github.com/qt/qtbase/blob/1dcd61d553e25c8986e7e6e65efacc51c3cf3bee/src/plugins/sqldrivers/sqlite/qsql_sqlite.cpp#L519-L534).

Interesting -- so Qt will have the unexpected behavior if you have multiple statements and don't call `numRowsAffected()` / `lastInsertId()` immediately after executing each one?